.travis/build.sh

```diff
@@ -47,19 +43,58 @@ fi
 
 echo "The job to configure OpenWhisk finished successfully"
 
-# Don't try and perform wsk actions the second it finishes deploying.
-# The CI ocassionaly fails if you perform actions to quickly.
-sleep 30
+# setup nginx
+pushd kubernetes/nginx
+  ./certs.sh localhost
+  kubectl -n openwhisk create configmap nginx --from-file=nginx.conf
+  kubectl -n openwhisk create secret tls nginx --cert=certs/cert.pem --key=certs/key.pem
+  kubectl apply -f nginx.yml
+
+  WSK_PORT=$(kubectl -n openwhisk describe service nginx | grep https-api | grep NodePort| awk '{print $3}' | cut -d'/' -f1)
+
+  # wait untill nginx is ready
+  TIMEOUT=0
+  TIMEOUT_COUNT=40
+  until $(curl --output /dev/null --silent -k https://localhost:$WSK_PORT) || [ $TIMEOUT -eq $TIMEOUT_COUNT ]; do
```

Comment:
This is a more definitive test to assure Nginx is running over a generic sleep (and hope).